node/node.go

```diff
@@ -170,73 +172,64 @@ type Node struct {
 	prometheusSrv    *http.Server
 }
 
-// NewNode returns a new, ready to go, Tendermint Node.
-func NewNode(config *cfg.Config,
-	privValidator types.PrivValidator,
-	nodeKey *p2p.NodeKey,
-	clientCreator proxy.ClientCreator,
-	genesisDocProvider GenesisDocProvider,
-	dbProvider DBProvider,
-	metricsProvider MetricsProvider,
-	logger log.Logger) (*Node, error) {
-
-	// Get BlockStore
+func getStoresForNewNode(config *cfg.Config, dbProvider DBProvider) (*bc.BlockStore, dbm.DB, error) {
```

Comment:
it's unclear what the second return param is.

```suggestion
func initDBs(config *cfg.Config, dbProvider DBProvider) (blockStoreDB *bc.BlockStore, stateDB dbm.DB, err error) {
```